UPGRADE-1.8.md

```diff
@@ -36,6 +42,10 @@
     +                   providers: [sylius_api_shop_user_provider, sylius_api_admin_user_provider]
             
             firewalls:
+                admin:
+                    remember_me:
```

Comment:
```suggestion
    +          admin:
    +              remember_me:
```

Reply:
Keys `admin `, `remember_me ` and `access_control` already exists, I think that we should and `+` to new lines